pkg/deploy/elbv2/tagging_manager.go

```diff
@@ -254,7 +259,12 @@ func (m *defaultTaggingManager) ListLoadBalancers(ctx context.Context, tagFilter
 		lbARNsWithinVPC = append(lbARNsWithinVPC, lbARN)
 		lbByARNWithinVPC[lbARN] = lb
 	}
-	tagsByARN, err := m.describeResourceTags(ctx, lbARNsWithinVPC)
+	tagsByARN := make(map[string]map[string]string)
+	if m.featureGates.Enabled(config.EnableRGTCalls) {
+		tagsByARN, err = m.describeResourceTagsViaRGT(ctx, services.ResourceTypeELBLoadBalancer, tagFilters)
+	} else {
```

Comment:
this is not correct.
the purpose of our change is to replace the entire client-side tag filter of `DescribeLoadBalancersAsList+ describeResourceTags ` to a single `RGT.GetResources` call.

With this change, we are still list all LoadBalancer & TargetGroups, which is not expected